source/docs/reference/environment-variables.md

```diff
@@ -0,0 +1,99 @@
+---
+id: environment-variables
+title: Environment variables
+---
+
+The environment variables available in Front-Commerce are the configurations that are likely to change depending on the current environment of your application. For instance, you could have three different environments: production, staging and local.
+
+These environment variables are the ones available on your server. You can also set them in your `.env` file in your root folder, and they will be loaded on launch.
```

Comment:
I think we should make it a bit clearer and make the difference between the build environment and the running server one. Also it would be a good idea to recall that one can access an environment variable under the `process.env` variable.

Finally, it could also be a good think to link to the last section or rewrite here that only `FRONT_COMMERCE_WEB_xxxxxx` variables are available in the React app.